python/py-upt/Portfile

```diff
@@ -2,50 +2,10 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           obsolete 1.0
 
 name                py-upt
+replaced_by         upt
 version             0.7
-revision            0
-
-maintainers         {@korusuke somaiya.edu:karan.sheth} openmaintainer
-description         Package software from any package manager to any distribution
-long_description    ${description}
-
-platforms           darwin
-supported_archs     noarch
-
-homepage            https://framagit.org/upt/upt
-master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
-distname            ${python.rootname}-${version}
-
+revision            1
 license             BSD
-checksums           sha256  7de9a0cc9d55c175b611ed748ee7ea572ec308411934769e2e647d1d26ba2eea \
-                    rmd160  6dc1b42ac5760f8225c81b1cd9539332b33b1b7c \
```

Comment:
the replacement currently only works for the ```py-upt``` stub port, not for ```py37-upt```. You should add an extra line here to explicitly specify that and ideally also add a date on when this port can be removed (typically one year after it's set to "replaced_by"): 
```
subport py37-upt "replaced_by upt"
# remove after June 17, 2020
```